kura/org.eclipse.kura.core/OSGI-INF/metatype/org.eclipse.kura.core.db.H2DbServer.xml

```diff
@@ -43,7 +43,7 @@
             description="Specifies the parameters for the server, see http://www.h2database.com/javadoc/org/h2/tools/Server.html for more details."/>
```

Comment:
I propose to add a note here, that port must be opened in the Firewall section - it's not opened automatically.